docs/getting-started/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Install Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/install/"
+---
+
+Shopify App CLI can be installed using a variety of package managers.
+
+### Brew (MacOS)
+
+You’ll need to run `brew tap` first to add Shopify’s third-party repositories to Homebrew.
+
+```console
+$ brew tap shopify/shopify
+$ brew install shopify-cli
+```
+
+### apt (Debian, Ubuntu)
```

Comment:
Maybe the heading should be "Debian-based Linux distributions (e.g., Debian, Ubuntu)"

Reply:
I'm opting to leave these organized by package manager first — I think people are more likely to recognize `apt` or `yum` when scanning the page. And we can make some educated guesses about the top distros to name-check and keep it more concise.